app/views/comments/_form.html.erb

```diff
@@ -76,7 +75,7 @@
             <!-- http://stackoverflow.com/questions/11235206/twitter-bootstrap-form-file-element-upload-button -->
             <label for="fileinput-choose-one-<%= comment_form_id %>">
               <input id="fileinput-choose-one-<%= comment_form_id %>" type="file" name="image[photo]" style="display:none;" />
-              <a onClick="handleClick()" style="cursor: pointer;" class="d-none d-md-inline text-underline"><%= translation('comments._form.choose_one') %></a>
```

Comment:
Deleting this function `handleClick`, similar to how I deleted `setInit`. Basically, I've decided that it's best not to `setState` when opening forms. For more info, check out [this write-up](https://github.com/publiclab/plots2/issues/9004#issue-784759413)